app/controllers/event_notes_controller.rb

```diff
@@ -30,32 +30,32 @@ def update
     event_note = EventNote.find(event_note_id)
 
     # Save the state of the existing event note.
-    previous_event_note_revision = EventNoteRevision.where(
-      event_note_id: event_note_id
-    ).order(
-      :version
-    ).last
-
-    if previous_event_note_revision
-      version = previous_event_note_revision.version + 1
+    event_note_revision = build_revision(event_note)
+
+    unless event_note_revision.save
+      render json: { errors: event_note_revision.errors.full_messages }, status: 422
```

Comment:
I think a doublerendererror could happen if the save fails - could do `return render json ...` to avoid